### PR TITLE
Added additional information to Tax Rates page

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added additional information to Tax Rates page: Tax titles for each storeview/Deleting tax rate

## Affected documentation pages

https://docs.magento.com/user-guide/tax/tax-zones-rates.html

### CODE REVIEW

File: src/tax/tax-zones-rates.md [hunk 1]
```diff
@@ -34,7 +34,15 @@ _Tax Zones and Rates_
 
    - Enter the **Rate Percent** that is used for the tax rate calculation.
 
-1. When complete, click <span class="btn">Save Rate</span>.
+1. If you have multiple stores, you can set Tax Titles for each store view.
+
+{:.bs-callout-info}
+Leave this field empty if you wish to use the tax identifier.
+
+![]({% link images/images/tax-rate-new-tax-titles.png %}){: .zoom}
```
Comment (dobooth) [inline src/tax/tax-zones-rates.md:42]: I don't think we need a new image here. Please remove it.

File: src/tax/tax-zones-rates.md [hunk 2]
```diff
@@ -52,3 +60,13 @@ _Tax Zones and Rates_
 1. Update the **Tax Titles** as needed.
 
 1. When complete, click <span class="btn">Save Rate</span>.
+
+## Delete Tax Rate
```
Comment (dobooth) [inline src/tax/tax-zones-rates.md:64]:
```suggestion
## Delete tax rate
```

**Review by dobooth — CHANGES_REQUESTED** [review]

**Comment by DmitryTsymbal** [comment]
Hello @dobooth , thank you for the review, I have applied the requested changes, please check it.
Thanks!

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@dobooth the Pull Request is successfully imported.

**Pull request merged**